gnomad/utils/transcript_annotation.py

```diff
@@ -205,3 +214,146 @@ def tissue_expression_ht_to_array(
         )
 
     return ht
+
+
+def preprocess_variants_for_tx(
+    ht: hl.Table,
+    filter_to_cds: bool = True,
+    filter_to_genes: Optional[List[str]] = None,
+    match_by_gene_symbol: bool = False,
+    filter_to_csqs: Optional[List[str]] = None,
+    ignore_splicing: bool = True,
```

Comment:
Are we sure this should default to True? They are in the context HT.

Reply:
This is the reason she put in her code: https://github.com/macarthur-lab/tx_annotation/blob/bea56ddf991607f7479358c0c80588f1a2ec9bc1/tx_annotation.py#L291

Reply:
When I look at a gene on the browser, it only has pext on the exons.

Reply:
I actually want to set it to "False", because I think if we spot check the coverage graph, the splice acceptor and donor are well covered.